packages/web-framework/bin/web-app-manager.js

```
@@ -254,11 +255,28 @@ class TiWebAppManager {
+    /**
+     * Used to process an application service request.
+     *
+     * @method
+     * @param {Object} session
+     * @param {string} service
+     * @param {Object} params
+     * @returns {Promise<Object>}
+     * @virtual
+     * @public
+     */
+    processServiceRequest( session, service, params ) {
```

ESLint

[error] 'params' is defined but never used.

(no-unused-vars)